tests/tests_assemble_workflow/data/artifacts/dist/opensearch-1.3.0/opensearch.txt

```diff
@@ -0,0 +1 @@
+This page intentionally left blank.
```

Comment:
what is this file for?

Reply:
Then we can say this file is a placeholder so people won't be confused?

Reply:
It already said ` This page intentionally left blank. `